composer.json

```diff
@@ -18,7 +18,7 @@
 		}
 	},
 	"require": {
-		"php": ">=7.2",
+		"php": ">=7.3",
```

Comment:
None of these changes actually *require* PHP 7.3 do they?

I wonder if it might not be better to keep the lower version so that it's easier for developers to get wsexport running locally (7.2 or earlier is still standard in lots of distros).

Reply:
Done.